mutagen/apev2.py

```diff
@@ -459,7 +459,7 @@ def save(self, filething):
         # "APE tags items should be sorted ascending by size... This is
         # not a MUST, but STRONGLY recommended. Actually the items should
         # be sorted by importance/byte, but this is not feasible."
-        tags.sort(key=len)
+        tags.sort(key=lambda tag: (len(tag), tag[8:256 + 8]))
```

Comment:
Shouldn't `(len(tag), tag)` also work?

Reply:
Yes it works too. I have chosen the slice because :
1) the tags will be ordered alphanumerically into the file (most people won't care anyway :)
2) this avoids having to parse a possibly huge tag value
But this may be less readable so it's up to you whether we should use the slice or not...

Reply:
I'd prefer it without the slice